examples/pipelines/guestbook.yaml

```diff
@@ -8,76 +8,92 @@ spec:
         - name: build-guestbook                  # 1.a Build guestbook go sample code.
           taskRef:
               name: build-push
-          sourceBindings:
-              - inputName: workspace
+          inputSourceBindings:
+              - name: workspace
                 sourceKey: guestbook
-          artifactStoreBindings:
-              - storeName: registry
-                storeKey: stagingRegistry
-                builtImage: gb-frontend          # TODO Add Commit SHA
+          outputSourceBindings:
+              - name: registry
+                sourceKey: stagingRegistry
           params:
               - name: pathToDockerfile
-                value: guestbook-go/Dockerfile
+                type: guestbook-go/Dockerfile
```

Comment:
something must have gone wrong here :thinking: this should definitely be value

Reply:
Yeah didn’t look right as type but should be easy enough to fix either in this PR or as a follow up